crates/db_schema/src/newtypes.rs

```diff
@@ -180,11 +180,22 @@ pub struct LtreeDef(pub String);
 pub struct DbUrl(pub(crate) Box<Url>);
 
 #[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Serialize, Deserialize, Default)]
-#[cfg_attr(feature = "full", derive(DieselNewType, TS))]
-#[cfg_attr(feature = "full", ts(export))]
+#[cfg_attr(feature = "full", derive(DieselNewType))]
 /// The report combined id
 pub struct ReportCombinedId(i32);
 
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Serialize, Deserialize, Default)]
+#[cfg_attr(feature = "full", derive(DieselNewType, TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// The person content combined id
+pub struct PersonContentCombinedId(i32);
+
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Serialize, Deserialize, Default)]
+#[cfg_attr(feature = "full", derive(DieselNewType, TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// The person saved combined id
+pub struct PersonSavedCombinedId(i32);
```

Comment:
According to the other PR these shouldnt have `derive(TS, Serialize, Deserialize)` nor ts(export)

Reply:
Got that fixed now.